mzbench/README.md

```diff
@@ -1 +1,58 @@
-Folder contains different testing scenarions for mzbench.
+# Scenarios
+
+Folder [scenarios](scenarios/) contains .bdl files ([Benchmark Definition Language](https://github.com/mzbench/mzbench/blob/master/doc/scenarios/spec.md)) that represents different testing scenarios for MZbench.
+
+
+List of scenarios that we will test.
+- [Fan-in](scenarios/fan_in.bdl)
+- [Fan-out](scenarios/fan_out.bdl)
+
+A prerequisite for running these test scenarios is to have Mainflux instance where you should:
+- create one Channel
+- create two Things
+- connect both Things to Channel
+
+Required paramaters are then set with running mzbench with --env param:
+
+`$ ./bin/mzbench run --env foo=bar --env n=42`
+
+or when using MZBench dashboard by importing and setting environment variables from scenario.
+
+
+| Environment variable | Description | Default |
+| --- | --- | --- |
+| MF_MZBENCH_MQTT_ENDPOINT | IP/domain of MQTT endpoint on Mainflux | 127.0.0.1 |
+| MF_MZBENCH_MQTT_PORT | Port of MQTT endpoint on Mainflux | 1883 |
+| MF_MZBENCH_CHANNEL_ID | ID of pre-provisioned channel on which both things are connected| |
+| MF_MZBENCH_THING_1_ID | First thing ID of pre-provisioned Mainflux thing | |
+| MF_MZBENCH_THING_1_KEY | First thing key of pre-provisioned Mainflux thing | |
+| MF_MZBENCH_THING_2_ID | Second thing ID of pre-provisioned Mainflux thing | |
+| MF_MZBENCH_THING_2_KEY | Second thing key of pre-provisioned Mainflux thing| |
+
+Scenarios have it's own parameters (like duration of test, rate of publishing...) that are set through environment variables, also. You can redefine an environment variable even during the benchmark without interrupting the benchmark itself with [change env](https://github.com/mzbench/mzbench/blob/master/doc/cli.md#change_env) command.
+
+## Fan-in
+
+In fan-in scenario numerous MQTT clients are sending messages on same Mainflux channel, but every client to exclusive subtopic on that channel. One subscriber is consuming all that messages.
```

Comment:
...all those messages.

Reply:
Can you pls link fan-in and fan-out, wiki links.